packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OpenBankingConnect/index.tsx

```diff
@@ -20,6 +20,12 @@ const Connect = (props: Props) => {
       )
     }
   }
+  useEffect(() => {
+    const { id } = props.order
+    if (id) {
+      props.simpleBuyActions.pollSBOrder(id)
+    }
+  }, [])
```

Comment:
actually we might do not need this if we solve this one https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts#L1323